kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaProtocolHandler.java

```diff
@@ -92,24 +96,43 @@ public void onLoad(NamespaceBundle bundle) {
             service.pulsar().getNamespaceService().getOwnedTopicListForNamespaceBundle(bundle)
                 .whenComplete((topics, ex) -> {
                     if (ex == null) {
+                        log.info("get owned topic list when onLoad bundle {}, topic size {} ", bundle, topics.size());
                         for (String topic : topics) {
                             TopicName name = TopicName.get(topic);
                             // already filtered namespace, check the local name without partition
                             if (Topic.GROUP_METADATA_TOPIC_NAME.equals(getKafkaTopicNameFromPulsarTopicname(name))) {
                                 checkState(name.isPartitioned(),
                                     "OffsetTopic should be partitioned in onLoad, but get " + name);
-                                KafkaTopicManager.removeLookupCache(name.toString());
 
                                 if (log.isDebugEnabled()) {
                                     log.debug("New offset partition load:  {}, broker: {}",
                                         name, service.pulsar().getBrokerServiceUrl());
                                 }
                                 groupCoordinator.handleGroupImmigration(name.getPartitionIndex());
                             }
+
+                            KafkaTopicManager.removeLookupCache(name.toString());
```

Comment:
Why move it outside the if block? By the way, I found this if condition didn't check the `name`'s namespace, it should be the same with `kafkaMetadataNs/kafkaMetadataNamespace`.

Reply:
Ok, I see the lookup cache now records not only `__consumer_offsets` but all topics now.